src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="top__bar"
+    action="#"
+    method="post">
```

Comment:
Refactor code in `index.html`. Check 5 point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix all cases
```suggestion
    <form
      class="top__bar"
      action="#"
      method="post"
    >
```